File: 03 - Metodos/mi_lib.py
class Operaciones:
    def __init__(self) -> None:
        pass

    @staticmethod
    def suma(lista, num = None):
        if num:
            result = []
            for element in lista:
                result.append(element + num)
        else:
            result = 0
            for element in lista:
                result = result + element
        return result

File: 03 - Metodos/mi_lib.js
class Operaciones {
    constructor() {}

    static suma(lista, num = null) {
        if (num) {
            let result = [];
            for (let element of lista) {
            result.push(element + num);
            }
            return result;
        } else {
            let result = 0;
            for (let element of lista) {
            result += element;
            }
            return result;
        }
    }
}

module.exports = {
    Operaciones: Operaciones
};

